stuffer/s2n_stuffer_text.c

```diff
@@ -45,19 +45,13 @@ int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer, uint32_t *skipp
 {
     POSIX_PRECONDITION(s2n_stuffer_validate(s2n_stuffer));
     uint32_t initial_read_cursor = s2n_stuffer->read_cursor;
-    while (s2n_stuffer->read_cursor < s2n_stuffer->write_cursor) {
-        switch (s2n_stuffer->blob.data[s2n_stuffer->read_cursor]) {
-        case ' ':              /* We don't use isspace, because it changes under locales */
-        case '\t':
-        case '\n':
-        case '\r':
-            s2n_stuffer->read_cursor += 1;
-            break;
-        default:
-            goto finished;
-        }
+    while (s2n_stuffer_data_available(s2n_stuffer)
```

Comment:
This looks like an unrelated change?

Reply:
It's basically a more readable replacement for `s2n_stuffer->read_cursor < s2n_stuffer->write_cursor`.